kinbot/reaction_generator.py

```diff
@@ -177,6 +177,7 @@ def generate(self):
                             ending = 'well'
                         sp_energy = self.qc.get_qc_energy('{}_{}'.format(str(self.species.chemid), ending))[1]
                         sp_zpe = self.qc.get_qc_zpe('{}_{}'.format(str(self.species.chemid), ending))[1]
+                        print("tse: {}, tsz: {} spe: {} spz: {}".format(ts_energy, ts_zpe, sp_energy, sp_zpe))
```

Comment:
this should go I think

Reply:
yes, i missed deleting this print line by mistake